src/node_file.cc

```diff
@@ -1219,7 +1219,7 @@ static void Read(const FunctionCallbackInfo<Value>& args) {
 
   char * buf = nullptr;
 
-  Local<Object> buffer_obj = args[1]->ToObject(env->isolate());
+  Local<Object> buffer_obj = args[1]->ToObject(env->context()).ToLocalChecked();
```

Comment:
I think this can be `.As<Object>()` as well since we already checked that it's a buffer

Reply:
You mean I should update as follow:

```cpp
Local<Object> buffer_obj = args[1]->ToObject(env->context()).ToLocalChecked();
```

to

```cpp
Local<Object> buffer_obj = args[1].As<Object>();
```

It is OK ?
I changed as described above, the test passed.

Reply:
Yes, exactly. If you know that something is an `Object` instance, I would recommend using `.As<Object>()`, because that boils down to a simple pointer cast (i.e. has zero runtime cost), unlike `ToObject()` which always calls into the JS engine.

Reply:
Thank you for confirmation.

> because that boils down to a simple pointer cast (i.e. has zero runtime cost), unlike ToObject() which always calls into the JS engine.

I didn't know that ! Thanks for letting me know.